### PR TITLE
PAAS-2151: fix and improve the check of module startup after installation

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2151

Short description:

### CODE REVIEW

File: mixins/jahia.yml [hunk 1]
```diff
@@ -860,20 +860,24 @@ actions:
...
+        - script: |-
+            return {"result": 0, "tryCount": [1,2,3,4,5,6,7,8,9,10]}
+        - forEach(response.tryCount):
+          - if ("${globals.moduleVersionState}" != "started"):
+            - cmd [proc]: 'echo "Try #${@i}..."'
+            - checkVersionOfModule:
+                moduleSymname: ${this.moduleSymname}
+                moduleVersion: ${this.moduleVersion}
+            - if ("${globals.moduleVersionState}" != "started"):
+              - cmd [proc]: 'sleep 5'
```
Comment (laurentfufu) [inline mixins/jahia.yml:876]:
```suggestion
              - sleep: 5000
```

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/jahia.yml [hunk 2]
```diff
@@ -860,20 +860,24 @@ actions:
...
-        - log: ${this.moduleSymname}/${this.moduleVersion} has been installed
-      - else:
-        - return:
-            type: error
-            message: "An error occurred during module startup."
+        - script: |-
+            return {"result": 0, "tryCount": [1,2,3,4,5,6,7,8,9,10]}
+        - forEach(response.tryCount):
+          - if ("${globals.moduleVersionState}" != "started"):
+            - cmd [proc]: 'echo "Try #${@i}..."'
```
Comment (laurentfufu) [inline mixins/jahia.yml:871]:
```suggestion
            - log: "Try #${@i}..."
```

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/jahia.yml (hunk 1 above)
Comment (adoux) [reply mixins/jahia.yml]: :man_facepalming: 
Fixed!

File: mixins/jahia.yml (hunk 2 above)
Comment (adoux) [reply mixins/jahia.yml]: :man_facepalming: 
Fixed!

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**